arch/cpu/cc26x0-cc13x0/rf-core/rf-core.c

```diff
@@ -746,8 +747,6 @@ cc26xx_rf_cpe1_isr(void)
 
   if(HWREG(RFC_DBELL_NONBUF_BASE + RFC_DBELL_O_RFCPEIFG) & IRQ_RX_BUF_FULL) {
     PRINTF("\nRF: BUF_FULL\n\n");
-    /* set a flag that the buffer is full*/
-    rf_core_rx_is_full = true;
```

Comment:
The definition and declaration of `rf_core_rx_is_full` in this file and the header can also be removed now?